pkg/apis/cnsoperator/config/cns.vmware.com_storagepolicyquotas.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     kind: StoragePolicyQuota
     listKind: StoragePolicyQuotaList
     plural: storagepolicyquotas
-    singular: storagepolicy
+    singular: storagepolicyquota
```

Comment:
I wonder how we missed this one earlier.  This should be auto-generated.

Reply:
yes, it is auto-generated